src/pages/FridgeDetailPage.js

```diff
@@ -81,15 +96,23 @@ export const FridgeDetailPageComponent = ({
         maximumDate={maximumDate}
       />
 
-      <Paper style={{ flex: 1 }} contentContainerStyle={{ flex: 1, marginTop: 20 }}>
+      <Paper height={300} contentContainerStyle={{ flex: 1, marginTop: 20 }}>
         <AfterInteractions>
-          <VaccineChart
-            breaches={[breaches[0]]}
-            minLine={minLine}
-            maxLine={maxLine}
-            minDomain={minDomain}
-            maxDomain={maxDomain}
-          />
+          <FlexRow alignItems="center">
+            <VaccineChart
+              breaches={[breaches[0]]}
+              minLine={minLine}
+              maxLine={maxLine}
+              minDomain={minDomain}
+              maxDomain={maxDomain}
+            />
+            <SensorStatus
+              isInHotBreach={isInHotBreach}
+              isInColdBreach={isInColdBreach}
+              isLowBattery={isLowBattery}
+              currentTemp={currentTemperature}
+            />
+          </FlexRow>
```

Comment:
I think this chart styles will change more soon a little bit to match the designs